docs/visual-basic/programming-guide/language-features/procedures/troubleshooting-procedures.md

```diff
@@ -9,36 +9,33 @@ helpviewer_keywords:
   - "procedures [Visual Basic], about procedures"
 ms.assetid: 525721e8-2e02-4f75-b5d8-6b893462cf2b
 ---
-# Troubleshooting Procedures (Visual Basic)
+# Troubleshooting procedures (Visual Basic)
 This page lists some common problems that can occur when working with procedures.  
   
-## Returning an Array Type from a Function Procedure  
- If a `Function` procedure returns an array data type, you cannot use the `Function` name to store values in the elements of the array. If you attempt to do this, the compiler interprets it as a call to the `Function`. The following example generates compiler errors.  
-  
- `Function allOnes(ByVal n As Integer) As Integer()`  
-  
- `For i As Integer = 1 To n - 1`  
-  
- `' The following statement generates a`   `COMPILER ERROR`  `.`  
-  
- `allOnes(i) = 1`  
-  
- `Next i`  
-  
- `' The following statement generates a`   `COMPILER ERROR`  `.`  
-  
- `Return allOnes()`  
-  
- `End Function`  
+## Returning an array type from a function procedure
+
+If a `Function` procedure returns an array data type, you cannot use the `Function` name to store values in the elements of the array. If you attempt to do this, the compiler interprets it as a call to the `Function`. The following example generates compiler errors.  
+  
+```vb
+Function allOnes(ByVal n As Integer) As Integer()`  
+   For i As Integer = 1 To n - 1  
+      ' The following statement generates a COMPILER ERROR.  
+      allOnes(i) = 1  
+   Next  
+
+   ' The following statement generates a COMPILER ERROR.  
+   Return allOnes()  
+End Function
```

Comment:
Add missing code block closure 
```suggestion
End Function
```
```

Reply:
The block is closed already. Ron just had an extra ` still left.

Reply:
Yes, he added it and resolved this issue 😄